ios/Classes/TiNfcNfcAdapterProxy.m

```diff
@@ -32,26 +39,100 @@ - (NFCNDEFReaderSession *)nfcSession
   return _nfcSession;
 }
 
+- (NFCTagReaderSession *)nfcTagReadersession
+{
+  if (_nfcTagReadersession == nil) {
+    _nfcTagReadersession = [[NFCTagReaderSession alloc] initWithPollingOption:NFCPollingISO14443 delegate:self queue:nil];
```

Comment:
Here you are polling for ```NFCPollingISO14443``` only. but there is many option values which can be used. Please create constants for all the options and get options from parameter so that user can send these from javascript. 
eg. 
```
- (NFCTagReaderSession *)nfcTagReadersession:(id)arg
{
    NSArray *pollingOptions = [type valueForKey:@"pollingOptions"];
    NSInteger pollingValue = 0;
    for (NSInteger value in pollingOptions) {
        pollingValue = pollingValue | value;
    }
  if (_nfcTagReadersession == nil) {
    _nfcTagReadersession = [[NFCTagReaderSession alloc] initWithPollingOption:pollingValue delegate:self queue:nil];
  }
  return _nfcTagReadersession;
}
```

Reply:
Updated code accordingly